inc/Engine/Media/Lazyload/CSS/Front/Extractor.php

```diff
@@ -149,6 +187,28 @@ protected function extract_urls( string $content, string $file_url = '' ): array
 		return $output;
 	}
 
+	/**
+	 * Reformat css
```

Comment:
It could be nice to have a more descriptive description of this function, in my opinion. 
When you read `reformat css` in the first place, you kinda wonder `reformat to what and why? `

Reply:
Updated, please check again